specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -258,7 +261,135 @@ sealed partial class App : Application
   }
 }
 ```
+## Registration of the Push Activator for unpackaged apps for protocol activation with PushNotificationsLongRunningTask
+The main difference here for unpackaged applications is that RegisterActivator will consume ProtocolActivator</br>
+and register with PushNotificationsLongRunningTask, a long running task that is shipped with PRmain, to handle</br>
+receiving raw notifications.
+
+**NOTE:** Packaged applications running on Windows builds earlier than 19041 will need to use ProtocolActivator</br>
+due to SetTaskEntryPointClsid() being unavailable on those builds.
+
+```cpp
+int main()
+{
+    // Take a dependency on Windows App SDK preview 
+    const UINT32 majorMinorVersion{ 0x00000008 }; 
+    PCWSTR versionTag{ L"preview" }; 
+    const PACKAGE_VERSION minVersion{};
+
+    const HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) }; 
+    // Check the return code. If there is a failure, display the result.
+    if (FAILED(hr)) 
+    { 
+        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n", 
+            hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision); 
+        return hr; 
+    }
+
+    PushNotificationRegistrationOptions pushOptions = PushNotificationManager::GetPushActivatorOptions(true);
```

Comment:
Might want to remove that version and bootstrap stuff in the top for this sample.

Reply:
Unless this is critical for the workings of our API in some way?

Reply:
All unpkg apps must call MddBootstrapInitialize/MddBootstrapShutdown so I figured our sample code should follow the WindowsAppSDK guidelines.

When an unpkg application exits, it removes its reference from DDLM. So that PRfwk can be serviced when all unpkg references are released.

Reply:
Also it adds the PRfwk to the packaged graph of the unpkg app which gives them access to our apis.

Reply:
Gotcha...might want to add a comment next to that to say this is critical code for packaged apps to function with the framework